server.js

```diff
@@ -28,22 +32,26 @@ var socketio = require('socket.io')(serverHTTPS, {
   serveClient: (config.env === 'production') ? false : true,
   path: '/socket.io-client'
 });
-app.all('*', function(req,res){
-	console.log('asdf');
+app.all('*', function(req,res, next){
+	if (!req.connection.encrypted) {res.redirect('https://sphereable.com')}
+	else {next();}
 })
 require('./server/config/socketio')(socketio);
 require('./server/config/express')(app);
 require('./server/routes')(app);
 
 // Start server HTTPS
-serverHTTPS.listen(config.port, config.ip, function () {
+serverHTTPS.listen(process.env.NODE_ENV === 'production' ? 443 : config.port, config.ip, function () {
   console.log('Express server listening on %d, in %s mode', config.port, app.get('env'));
 });
 
-serverHTTP.listen(80, config.ip, function () {
-  // console.log(req.connection.encrypted?true:false);
-  console.log('Express server listening on %d, in %s mode', config.port, app.get('env'));
-});
+if (process.env.NODE_ENV === 'production') {
+	serverHTTP.listen(80, config.ip, function () {
+	  // console.log(req.connection.encrypted?true:false);
```

Comment:
Fix indentation, may be check if Sublime settings is set to tabs rather than spaces.